packages/app-desktop/gui/NoteEditor/utils/contextMenu.test.ts

```diff
@@ -38,4 +38,25 @@ describe('contextMenu', () => {
 		}
 		console.error = consoleError;
 	});
+
+	it('should generate resource info', async () => {
+		const opts: ContextMenuOptions = {
+			itemType: ContextMenuItemType.Image,
+			resourceId: '',
+			mime: 'image/png',
+			filename: 'testfile.png',
+			linkToCopy: null,
+			textToCopy: null,
+			htmlToCopy: null,
+			insertContent: function() { },
+		};
+		const result: ResourceInfo = {
+			resource: null,
+			resourcePath: null,
+			filename: 'testfile.png',
+		};
+
+		const info = await resourceInfo(opts);
+		expect(info).toEqual(result);
+	});
```

Comment:
Sorry I feel this only tests the structure of the function output, and we can assume this is now correct because types are enabled. So please could you remove these tests? To be useful you'd have to create resources, etc. but let's skip that for now.